src/NetTopologySuite/GeometriesGraph/QuadrantOp.cs

```diff
@@ -11,8 +11,8 @@ namespace NetTopologySuite.GeometriesGraph
     /// 2 | 3
     /// </para>
     /// </summary>
-    [Obsolete("Use NetTopologySuite.Topology.Quadrant and NetTopologySuite.Topology.QuadrantExtensions")]
-    public class QuadrantOp
+    [Obsolete("Use NetTopologySuite.Geometries.Quadrant and NetTopologySuite.Geometries.QuadrantExtensions")]
+    public static class QuadrantOp
```

Comment:
This is not a breaking change?

Reply:
It shouldn't be -- since `QuadrantOp` can't be instantiated, the only thing you can do with it is call static methods.

I guess it's theoretically possible that someone's doing something like this:
```csharp
QuadrantOp op = null;
```

but that's really far-fetched.

Reply:
ahh, the API Compat check actually rejects it, so I'm going to revert.